CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+related that could be of help.
+
+When opening a new issue, make sure to fill the issue template. They are made
+to make the subject to be as understandable as possible, not doing so may result 
+in your issue not being managed right away, if you don't understand something
+(be it regarding your own problem/the issue template/the library), please state 
+so.
+
+## Development
+Note before contributing that everything done here is under the [MIT](https://opensource.org/license/mit/) license.
+### Naming
+Naming is an important matter, and it shouldn't be changed unless necessary.
+
+Game **names** should be added as they appear on steam (or other storefront 
+if not listed there) with the release year appended in brackets (except when the 
+release year is already part of the name).  
+If there is a mod that needs to be added (or it adds the support for server 
+queries for the game), its name should be composed of the game name, a separating
+**bracket**, the mod name and the release year as specified previously
+(e.g. `Grand Theft Auto V - FiveM (2013)`).
+
+A game's **identification** is lowercased only and be created following these rules:
+1. Names composed of a maximum of two words (unless #4 applies) will result in an 
+id where the words are concatenated (`Dead Cells` -> `deadcells`), acronyms in 
+the name count as a single word (`S.T.A.L.K.E.R.` -> `stalker`).
+2. Names of more than two words shall be made into an acronym made of the 
+initial letters (`The Binding of Isaac` -> `tboi`), [hypenation composed words](https://prowritingaid.com/hyphenated-words) 
+don't count as a single word, but of how many parts they are made of 
+(`Dino D-Day`, 3 words, so `ddd`).
+3. If a game has the exact name as a previously existing id's game 
+(`Star Wars Battlefront 2`, the 2005 and 2017 one), append the release year to
+the newer id (2005 would be `swbf2` (suppose we already have this one supported) 
+and 2017 would be `swbf22017`).
+4. If a new id (`Day of Dragons` -> `dod`) results in an id that already exists 
+(`Day of Defeat` -> `dod`), then the new name should ignore rule #2 
+(`Day of Dragons` -> `dayofdragons`).
+5. Roman numbering will be converted to arabic numbering (`XIV` -> `14`).
+6. Unless numbers are at the end of a name, they will be considered words, 
+but digits will always be used instead of the acronym (counter to #2) (`Left 4 
+Dead` -> `l4d`) unless they at the start position (`7 Days to Die` -> `sdtd`), 
+if they are at the end (such as sequel number or the year), always append them 
+(`Team Fortress 2` -> `teamfortress2`, `Unreal Tournament 2003` -> 
+`unrealtournament2003`).
+7. If a game has multiple id entries (by specifying the edition (Minecraft: 
+Java and Bedrock) or the release version (Counter-Strike 1: 1.5 and 1.6)), one
+more entry by the base name (`minecraft` and `counterstrike` respectively) can
+be added, where it queries in a group said included entries.
```

Comment:
This mentions the case for the auto query but doesn't tell us what to do for individual protocols. Should we stop adding ids for individual protocols e.g. `minecraftjava`?

Or should that already be included as a word anyway. I have been leaving out because the protocol is wrapped in `()`, which is similar to how the year is extra info, but games could included `()` in their titles anyway.

Perhaps adding the protocol should be specified in name rules.

Reply:
Modified, what do you think?